### PR TITLE
Add single commission card view

### PR DESCRIPTION
**Pull request by ghost1227**
First round refresh for #148

### CODE REVIEW

File: includes/admin/class.admin-notices.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<?php
+
+// Exit if accessed directly
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+class EDDC_Admin_Notices {
```
Comment (pippinsplugins) [inline includes/admin/class.admin-notices.php]: Let's please get PHPdoc added to this class.

Comment (ghost1227) [inline includes/admin/class.admin-notices.php]: @pippinsplugins done

File: includes/admin/class.admin-notices.php [hunk 2]
```diff
@@ -1,10 +1,28 @@
...
 if ( ! defined( 'ABSPATH' ) ) {
 	exit;
 }
 
+
+/**
+ * EDDC_Admin_Notices Class
+ *
+ * @since 3.3
+ */
```
Comment (pippinsplugins) [inline includes/admin/class.admin-notices.php]: We also need to do all methods within the class

Comment (ghost1227) [inline includes/admin/class.admin-notices.php]: @pippinsplugins fixed, my bad

**Pull request merged**